game/src/main/java/net/demilich/metastone/game/spells/desc/source/CardSource.java

```diff
@@ -71,6 +75,10 @@ public CardList getCards(GameContext context, Entity source, Player player) {
 					.values());
 		}
 
+		if (getDesc().getBool(CardSourceArg.INVERT)) {

```

Comment:
I'm not sure if it's safe to do this. You shouldn't have to remember orders in the game, so maybe the `Akarador` card should just get its own target and not get this special logic.